javascript/ql/src/Security/CWE-400/PrototypePollutionUtility.ql

```diff
@@ -113,11 +120,57 @@ class EntriesEnumeratedPropName extends EnumeratedPropName {
   }
 }
 
+/**
+ * Gets a function that enumerates object properties when invoked.
+ *
+ * Invocations takes the following form:
+ * ```js
+ * fn(obj, (value, key, o) => { ... })
+ * ```
+ */
+SourceNode propertyEnumerator() {
+  result = moduleImport("for-own") or
+  result = moduleImport("for-in") or
+  result = moduleMember("ramda", "forEachObjIndexed") or
+  result = LodashUnderscore::member("forEach") or
```

Comment:
If the evaluation turns out fine, we should consider adding jQuery's forEach..

This particular predicate can probably be extended with a thousand more cases though...